README.md

```diff
@@ -109,10 +109,6 @@ your project (for example, at `bin/venv-update`).
 Then, apply a change like this to your `tox.ini` file:
 
 ```patch
-  [tox]
-  envlist = py27,py34
-+ skipsdist = true
-
```

Comment:
But nobody's using tox1 anymore are they? jk =/

IMHO that's a special case in terms of the users of the README, but it's a fair point. I'll fix it.